apps/web/src/utils/is-ha-users.ts

```diff
@@ -0,0 +1,9 @@
+import { Authorities, EmployeeRO, RoleSlug } from '@ien/common';
+
+export const isHaUsers = (user?: EmployeeRO): boolean => {
+  return (
+    !!user &&
+    user.roles.every(role => role.slug !== RoleSlug.Admin) &&
+    user.organization !== Authorities.HMBC.name
```

Comment:
what if the user is MoH?

Reply:
MoH has only reporting role by default so then don't have higher privileges than HA.